src/ai/backend/client/func/session.py

```diff
@@ -126,7 +126,8 @@ async def paginated_list(
                 "filter": (filter, "String"),
                 "order": (order, "String"),
             },
-            fields,
+            # fields,
+            [session_fields["session_id"], session_fields["agent_ids"]],
```

Comment:
This removes all the default fields in the `backend.ai ps` command! Please include `agent_ids` inside `_default_list_fields`.

Reply:
thanks! my mistake..